rapids/README.md

```diff
@@ -0,0 +1,96 @@
+# RAPIDS
+
+This initialization action installs the latest release version of [RAPIDS](https://rapids.ai/) on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster. 
+
+These initialization actions automate the process of setting up a Dask-cuDF cluster by running the following:
+
+On the Dataproc master node:
+- `dask-scheduler`, and `dask-cuda-worker`
+
+On the Dataproc worker nodes:
+- `dask-cuda-worker`
+
+The master Cloud Dataproc node will run the dask-scheduler _and_ dask-cuda-workers. All Cloud Dataproc workers will run dask-cuda-workers.
+
+Our initialization actions do the following:
+1. [install nvidia GPU driver](install-gpu-driver.sh)
+2. [install RAPIDS](install-rapids.sh) - [installs miniconda](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/tree/master/conda), and [conda packages](conda-environment.yml)
+3. [start dask-scheduler and dask-cuda-workers](launch-dask.sh)
+
+## Using this initialization action
+You can use this initialization action to create a new Dataproc cluster with RAPIDS installed:
+
+1. Using the `gcloud` command to create a new cluster with this initialization action. The following command will create a new cluster named `<CLUSTER_NAME>`.
+
+    ```bash
+    DATAPROC_BUCKET=dataproc-initialization-actions
+
+    gcloud beta dataproc clusters create <CLUSTER_NAME> \
+    --zone us-east1-c \
+    --master-accelerator type=nvidia-tesla-t4,count=4 \
```

Comment:
does this only work with `nvidia-tesla-t4` type of accelerator? In any case (yes or no), can you clarify that in the documentation below?

Also, note, nvidia-tesla-t4 is not documented as a supported accelerator at: https://cloud.google.com/dataproc/docs/concepts/compute/gpus

Reply:
RAPIDS also works on Tesla P100 and V100 GPU types. It will not work on K80 types.

The user currently must specify the correct driver URL, as drivers are specific to GPU type. I checked but did not see an environment property init-scripts can access that tells what GPU type is attached. If there is one you can point me to, I can look at updating install-gpu-driver.sh to automatically pick the right one.